src/runtime/GlobalObjectBuiltinString.cpp

```diff
@@ -307,6 +307,52 @@ static Value builtinStringRepeat(ExecutionState& state, Value thisValue, size_t
 
 static Value builtinStringReplace(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
 {
+#ifdef ESCARGOT_ENABLE_ES2015
+    ASSERT(argv != nullptr);
```

Comment:
Please move it to above line which is ``#ifdef ESCARGOT_ENABLE_ES2015`` so to match all conditions.

Reply:
I think this is incorrect, with 0 argument the argv possibly can be `nullptr`. The correct way to the assert is:

`ASSERT(argc == 0 || argv != nullptr)`